src/main.zig

```diff
@@ -439,6 +439,8 @@ const usage_build_generic =
     \\  -fno-unwind-tables        Never produce unwind table entries
     \\  -fLLVM                    Force using LLVM as the codegen backend
     \\  -fno-LLVM                 Prevent using LLVM as the codegen backend
+    \\  -flibLLVM                 Force using LLVM shared library apias the codegen backend
```

Comment:
s/apias/as

Reply:
Would you mind making the new ones lower case? `-fLLVM` and `-fno-LLVM` are the only non-lower case flags - apologies for that - and I plan to rename them as well.

Reply:
haha, fair! feel free to leave it as-is, I'll change them all at once just before the release.